lib/manager/common.ts

```diff
@@ -172,6 +172,9 @@ export interface PackageDependency<T = Record<string, any>> extends Package<T> {
   autoReplaceStringTemplate?: string;
   depIndex?: number;
   editFile?: string;
+  separateMinorPatch?: boolean;
+  separateMajorMinor?: boolean;
+  separateMultipleMajor?: boolean;
```

Comment:
Actually, I don't think it's better way to make these fields availabe as `extract.ts` results, but the code already can deal with it.

Reply:
Do we need them all now, or just minorpatch?

Reply:
I'm thinking about extracting these three options into separate interface, and them mix it into both `PackageDependency` and `LookupUpdateConfig`